cypress/support/e2e.js

```diff
@@ -1,20 +1,5 @@
-// ***********************************************************
-// This example support/e2e.js is processed and
-// loaded automatically before your test files.
-//
-// This is a great place to put global configuration and
-// behavior that modifies Cypress.
-//
-// You can change the location of this file or turn off
-// automatically serving support files with the
-// 'supportFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/configuration
-// ***********************************************************
-
-// Import commands.js using ES2015 syntax:
 import "./commands";
 
-// Alternatively you can use CommonJS syntax:
-// require('./commands')
+Cypress.on("uncaught:exception", (err, runnable) => {
+  return false;
+});
```

Comment:
O que isso faz @Akellymourab ?

Reply:
ele vai ignorar o type error do console e continuar o teste.

no meu local, se rodar os testes de navbar, no meio do teste ele estoura esse erro:

```
The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.

> Cannot read properties of null (reading 'name')

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the uncaught:exception event.Learn more  
```

já no ambiente local do Henrrique, rodou normalmente nem estourar erro algum.